### DISCUSSION TITLE
Possible to install older versions of SO?

### DISCUSSION THREAD

**Discussion by ezeadam123** [Security-Onion-Solutions/securityonion#5217] [Unsupported Versions] [answered]
Is it possible to install older versions of SO? The newer versions are not compatible with my device.. I do not have the min. amount of RAM, and a broke college student.

**Comment by dougburks** [comment, marked as answer]
We do not recommend older versions of Security Onion as they are End Of Life and no longer supported by us or by Ubuntu.

You should be able to do an Import installation of Security Onion 2 in only 4GB RAM.  Just follow the screenshot tour here:
https://blog.securityonion.net/2021/08/security-onion-2370-now-available.html
